src/framework/uicomponents/view/internal/platform/macos/macospopupviewclosecontroller.mm

```diff
@@ -61,3 +70,14 @@
 
     return false;
 }
+
+void MacOSPopupViewCloseController::initWindowMinimizedObserver()
+{
+    minimizeObserverToken = [[NSNotificationCenter defaultCenter]
+                             addObserverForName:@"NSWindowWillMiniaturizeNotification"
+                             object:nil

```

Comment:
By not providing an `object` here, you will receive minimise notifications from all windows, not just the main window. Is that intentional?

Reply:
Fixed!